tests/install_upgrade_operators/console_cli_download/conftest.py

```diff
@@ -27,6 +32,17 @@ def virtctl_console_cli_downloads_spec_links(admin_client):
     return get_console_spec_links(admin_client=admin_client, name=VIRTCTL_CLI_DOWNLOADS)
 
 
+@pytest.fixture(scope="class")
+def virtctl_console_cli_downloads_spec_links_scope_class(admin_client):
+    """
+    Get console cli downloads spec links
+
+    Returns:
+        ConsoleCLIDownload instance.spec.links
+    """
+    return get_console_spec_links(admin_client=admin_client, name=VIRTCTL_CLI_DOWNLOADS)
```

Comment:
I would recomend to return list here, `[url.href for url in ...` 
it is the only way how you use this fixture, so no point for return the ConsoleCLIDownload